include/envoy/network/udp_packet_writer_handler.h

```diff
@@ -0,0 +1,145 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+
+#include "envoy/api/io_error.h"    // IoCallUint64Result
+#include "envoy/buffer/buffer.h"   // Buffer
+#include "envoy/network/address.h" // Address
+#include "envoy/network/socket.h"  // Socket
+#include "envoy/stats/scope.h"
+#include "envoy/stats/stats_macros.h"
+
+namespace Envoy {
+namespace Network {
+
+static const uint64_t K_MAX_OUTGOING_PACKET_SIZE = 1452; // Based on quic::kMaxOutgoingPacketSize
+
+#define UDP_PACKET_WRITER_STATS(GAUGE)                                                             \
```

Comment:
It's very important to know how many packets are sent in a batch. How about add a HISTOGRAM for that?

Reply:
Thanks for the suggestion! 
Added the `HISTOGRAM(pkts_sent_per_batch, Unspecified)` here